### ISSUE TITLE
What does ‘lineitems.id’ represent？

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9605]
![下载](https://github.com/user-attachments/assets/2f72d5fb-09c4-417e-84d0-3b683bc4c8ae)
What does the value returned to us by this field ‘lineitems.id’ in this endpoint（https://api.{environment}/order/checkout-forms） represent? Is it the manufacturer code（the seller's internal SKU code）we entered when creating the offer?

**Comment by Lukasz-Zurek** [comment]
@yangcen6 No, **"lineItems.id"** relates to the information about items included in the order. So if the buyer bought many different products in one order - then we return lineItems for every different product in the order.

There is no field responsible for manufacturer's code in the orders, in our API.

**Comment by yangcen6** [comment]
What does it actually represent? EAN or？

**Comment by Lukasz-Zurek** [comment]
@yangcen6 If you mean **"lineItems.id"** array, you can check it closely for these specific resources [GET /order/checkout-forms](https://developer.allegro.pl/documentation/#operation/getListOfOrdersUsingGET) and [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation/#operation/getOrdersDetailsUsingGET). [Here](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#order-details) you can check the structure.
